config/commit-template

```diff
@@ -0,0 +1,5 @@
+The title
```

Comment:
What about:

```

Description:

Reviewers: @getsentry/team
```

My reasoning being, this commit template provides nothing. I'd be just deleting everything every time. I'd rather have prompts to fill in.